package.json

```diff
@@ -9,6 +9,7 @@
     "eject": "expo eject"
   },
   "dependencies": {
+    "@react-native-community/async-storage": "^1.12.1",
```

Comment:
It looks like you can remove this dependency as you've already go the newer `async-storage` library included below

Reply:
Done